### ISSUE TITLE
mono needed for Ionide language services

### ISSUE THREAD

**Issue by ctaggart** [ctaggart/dotnet-vscode#2]
https://github.com/ionide/ionide-vscode-fsharp/issues/66#issuecomment-220771163

When will .NET Core only be needed for language services?

**Comment by sivabudh** [comment]
Not adding anything to the discussion, but just want to say:

I hope it's soon!

**Comment by ctaggart** [comment]
https://github.com/mono/docker/blob/master/4.2.3.4/Dockerfile

```
RUN apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv-keys 3FA7E0328081BFF6A14DA29AA6A19B38D3D831EF

RUN echo "deb http://download.mono-project.com/repo/debian wheezy/snapshots/4.2.3.4 main" > /etc/apt/sources.list.d/mono-xamarin.list \
    && apt-get update \
    && apt-get install -y mono-devel ca-certificates-mono fsharp mono-vbnc nuget \
    && rm -rf /var/lib/apt/lists/*
```

https://packages.debian.org/jessie/mono-devel
https://packages.debian.org/jessie/mono-complete

mono-complete depends on mono-devel
https://packages.debian.org/jessie/mono-vbnc, Mono Visual Basic Compiler (VB.NET)

2014-11 http://www.jimmybyrd.me/Get-Latest-Mono-On-Debian/

**Comment by ctaggart** [comment]
fixed by #4
